components/EliminatedStatus/EliminatedStatus.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+/**
+ * @returns EliminatedStatus component
```

Comment:
How did you get by without adding a description of the function? I might have to add a rule if it's not requiring a description.

Reply:
I'm not sure, I have no warning or error in my console looking for a function description currently.
![CleanShot 2024-06-18Code001215@2x](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/c10db26b-8c2a-4119-9047-abddcf30e89c)

Reply:
Got it. Fixing this and adding rules.